src/test/java/com/bravo/user/controller/PaymentControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package com.bravo.user.controller;
+
+import com.bravo.user.App;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.service.PaymentService;
+import com.bravo.user.utility.PageUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ContextConfiguration(classes = {App.class})
+@ExtendWith(SpringExtension.class)
+@SpringBootTest()
+@AutoConfigureMockMvc
+class PaymentControllerTest {
+
+  final static String RESOURCE_URL = "/payment/retrieve/";
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @MockBean
+  private PaymentService paymentService;
+
+  private List<PaymentDto> payments;
+
+  @BeforeEach
+  public void beforeEach(){
+    final List<Integer> ids = IntStream
+        .range(1, 10)
+        .boxed()
+        .collect(Collectors.toList());
```

Comment:
Nitpick here.  Why is this better than a simple for loop or a List.of?

Reply:
Clean functional code, and was following conventions in other tests.

Even though a loop may be more performant at small numbers, I don't enough of a reason in a test class to make that change.

Reply:
The other tests are wrong :) .  Bike shedding is a real thing so I very much see that viewpoint.  In this case, I do prefer List.of if only because the values are explicit, but there are usually more important things to do.